WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -420,6 +436,60 @@ local methods = {
             end
         end
 
+        function self.callbacks.OnDragStart(_, _)
+            if WeakAuras.IsImporting() or not data.parent then return end;
+
+            data.dragging = true
+
+            self.frame:StartMoving()
+            self.frame:SetUserPlaced(false)
+        end
+
+        function self.callbacks.OnDragStop(_)
+            if data.dragging then
+                self.frame:StopMovingOrSizing()
+
+                local id, parent = data.id, data.parent
+                local parentData = WeakAuras.GetData(parent)
+                local children = parentData.controlledChildren
+
+                local i
+                for key, value in pairs(children) do
+                    if value == id then
+                        i = key; break
+                    end
+                end
+                if i then
+                    if IsOverDisplay(parent) then
+                        tremove(children, i)
+                        tinsert(children, 1, id)
+                        WeakAuras.Add(parentData);
+                        UpdateGroupOrder(children)
+                    else
+                        local j
+                        for key, value in pairs(children) do
+                            if key ~= i and IsOverDisplay(value) then
+                                j = key
+                                break
+                            end
+                        end
+                        if j then
+                            tremove(children, i)
+                            tinsert(children, j+1, id)
+                            WeakAuras.Add(parentData);
+                            UpdateGroupOrder(children)
+                        end
+                    end
+                else
+                    error("Display thinks it is a member of a group which does not control it")
+                end
+                WeakAuras.SortDisplayButtons()
+                WeakAuras.UpdateDisplayButton(parentData)
+
+                data.dragging = false
```

Comment:
Looking at this code, it looks to me that adding dragging between groups wouldn't be hard to add, right?